tests/XrdCl/GTestXrdHelpers.hh

```diff
@@ -0,0 +1,21 @@
+/** Contains some of the macros found in CppUnitXrdHelpers.hh,
+ *  converted into GTest equivalent. 
+ * 
+ *  Created by Angelo Galavotti in 2023.
+ */
```

Comment:
See header in `XrdClPoller.cc`, new files should use that format for the top header.